src/main/java/de/sybit/codingcamp2019/service/GameServiceImpl.java

```diff
@@ -103,6 +103,8 @@ public GameStateEnum checkGameStatus(@NotNull HttpSession session, @NotNull PinP
    public void restartGame(HttpSession session) {
       LOGGER.debug("--> restartGame");
 
+      createGameFor(session);
```

Comment:
In dieser Methode muss lediglich das Session Attribut `SessionKeys.SESSION_GAME.toString()` gelöscht werden und nicht die Methode `createGameFor(session)`

Reply:
erledigt